datasets/ohsumed/ohsumed.py

```diff
@@ -0,0 +1,167 @@
+isbn="978-1-4471-2099-5"
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+The OHSUMED test collection is a set of 348,566 references from
+MEDLINE, the on-line medical information database, consisting of
+titles and/or abstracts from 270 medical journals over a five-year
+period (1987-1991). The available fields are title, abstract, MeSH
+indexing terms, author, source, and publication type.
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "http://davis.wpi.edu/xmdv/datasets/ohsumed.html"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = "CC BY-NC 4.0"
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URLs = {"ohsumed": "https://storage.googleapis.com/corona-tweet/ohsumed.zip"}
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class Ohsumed(datasets.GeneratorBasedBuilder):
+    """TODO: Short description of my dataset."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    # This is an example of a dataset with multiple configurations.
+    # If you don't want/need to define several sub-sets in your dataset,
+    # just remove the BUILDER_CONFIG_CLASS and the BUILDER_CONFIGS attributes.
+
+    # If you need to make complex sub-parts in the datasets with configurable options
+    # You can create your own builder configuration class to store attribute, inheriting from datasets.BuilderConfig
+    # BUILDER_CONFIG_CLASS = MyBuilderConfig
+
+    # You will be able to load one or the other configurations in the following list with
+    # data = datasets.load_dataset('my_dataset', 'first_domain')
+    # data = datasets.load_dataset('my_dataset', 'second_domain')
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="ohsumed", version=VERSION, description="This part of my dataset covers a first domain"
+        )
+    ]
+
+    def _info(self):
+        # TODO: This method specifies the datasets.DatasetInfo object which contains informations and typings for the dataset
+        features = datasets.Features(
+            {
+                "seq_id": datasets.Value("string"),
+                "medline_ui": datasets.Value("string"),
+                "mesh_terms": datasets.Value("string"),
```

Comment:
Can this be a `ClassLabel` feature ?

Reply:
the number of `mesh_terms` is very high, won't be practical to add it as a class label.